src/SFML/Window/WindowImpl.cpp

```diff
@@ -208,7 +208,7 @@ void WindowImpl::processJoystickEvents()
                     Joystick::Axis axis = static_cast<Joystick::Axis>(j);
                     float prevPos = m_previousAxes[i][axis];
                     float currPos = m_joystickStates[i].axes[axis];
-                    if (fabs(currPos - prevPos) >= m_joystickThreshold)
+                    if (std::fabs(currPos - prevPos) >= m_joystickThreshold)
```

Comment:
We should use `std::abs()` directly, as it's overloaded for `float` in C++.
This would be consistent as we don't use `std::sinf()` and other explicit C float variants, either.

Reply:
Right, I somehow thought the float overload was only provided in C++11 and even checked cppreference, but apparently I can't read. 😄

Reply:
It certainly doesn't help that [the `abs` page](https://en.cppreference.com/w/cpp/numeric/math/abs) only lists the integer overloads, and the float overloads are listed on [the `fabs` page](https://en.cppreference.com/w/cpp/numeric/math/fabs). I also had to check twice...